recipes/xorg-proto/all/conanfile.py

```diff
@@ -0,0 +1,86 @@
+from conans import AutoToolsBuildEnvironment, ConanFile, tools
+import glob
+import os
+import re
+import yaml
+
+required_conan_version = ">=1.33.0"
+
+
+class XorgProtoConan(ConanFile):
+    name = "xorg-proto"
+    description = "This package provides the headers and specification documents defining " \
+        "the core protocol and (many) extensions for the X Window System."
+    topics = ("conan", "xproto", "header", "specification")
+    license = "X11"
+    homepage = "https://gitlab.freedesktop.org/xorg/proto/xorgproto"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "arch", "compiler", "build_type"
+
+    generators = "pkg_config"
+
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _settings_build(self):
+        return self.settings_build if hasattr(self, "settings_build") else self.settings
```

Comment:
maybe overkill if used in build() only? You could replace by your own simplification: `getattr(self, "settings_build", self.settings)`

Reply:
Indeed overkill.
But it doesn't hurt, I think.
```suggestion
    @property
    def _settings_build(self):
        return getattr(self, "settings_build", self.settings)
```